src/tests/ftest/pool/create_all_vm.py

```diff
@@ -129,6 +129,28 @@ def test_rank_filter(self):
             abs(scm_unused_bytes - self.scm_avail_bytes), delta_bytes,
             msg.format(scm_unused_bytes, self.scm_avail_bytes, delta_bytes))
 
+    def get_recycle_pools_delta_bytes(self, pool_count):
```

Comment:
You might want to add description for `pool_count`. Or you could move line 169 inside this method.

Reply:
- [ ] Add method description